slam-libraries/viam-cartographer/src/slam_service/config.cc

```diff
@@ -67,6 +70,24 @@ void ParseAndValidateConfigParams(int argc, char** argv,
     slamService.path_to_data = FLAGS_data_dir + "/data";
     slamService.path_to_map = FLAGS_data_dir + "/map";
     slamService.configuration_directory = FLAGS_data_dir + "/config/lua_files";
+
+    // Find the lua files.
+    path pathToLuasFromConfig(slamService.configuration_directory);
```

Comment:
This code looks for lua files being placed in our appimage recipe, so users that have an appimage do not need to place the files into their config directory